src/browser-providers/local.ts

```diff
@@ -19,13 +27,34 @@ export class LocalBrowserProvider extends BrowserProvider<Browser> {
     this.session = browser;
     return this.session;
   }
+
   async close(): Promise<void> {
     return await this.session?.close();
   }
+
   public getSession() {
     if (!this.session) {
       return null;
     }
     return this.session;
   }
+
+  public async getContext(
```

Comment:
Can you also add this method on the abstract base class that they inherit from ?